packages/common/src/hardhat/index.js

```diff
@@ -0,0 +1,6 @@
+const fs = require("fs");
+const path = require("path");
+
+["tasks"].forEach((folder) =>
```

Comment:
I think this syntax is great for this use case, since hardhat will not be exported to frontends importing this package.

We should be aware of where we use it, thought, as it isn't compatible with frontends.

Reply:
Ah, because `fs` and `path` don't exist on frontends?